pkg/standalone-utils/contracts/ERC4626CowSwapFeeBurner.sol

```diff
@@ -52,19 +55,34 @@ contract ERC4626CowSwapFeeBurner is CowSwapFeeBurner {
         IERC20 feeToken,
         uint256 exactFeeTokenAmountIn,
         IERC20 targetToken,
-        uint256 minTargetTokenAmountOut,
+        uint256 encodedMinAmountsOut,
         address recipient,
         uint256 deadline
     ) external override onlyProtocolFeeSweeper nonReentrant {
         // In this case we first pull the wrapped token, unwrap, and then proceed to burn by creating an order for
         // the underlying token.
         feeToken.safeTransferFrom(msg.sender, address(this), exactFeeTokenAmountIn);
 
-        // Redeem and overwrite inputs with new asset and unwrapped amount.
+        uint256 minTargetTokenAmountOut = uint128(encodedMinAmountsOut >> 128);
+        uint256 minERC4626AmountOut = uint128(encodedMinAmountsOut & type(uint128).max);
```

Comment:
Not sure if it's worth it, but a function to decode the minAmountsOut in target and ERC4626 would make this part of the code clearer.